go-controller/pkg/ovn/libovsdbops/portgroup.go

```diff
@@ -271,10 +271,10 @@ func DeleteACLsFromPortGroupOps(nbClient libovsdbclient.Client, ops []libovsdb.O
 	uuids := make([]string, 0, len(acls))
 	for _, acl := range acls {
 		err := findACL(nbClient, acl)
-		if err != nil {
-			return nil, err
+		// Proceed to check others if we don't find an ACL.
```

Comment:
I _think_ this is wrong, we still wan't to check the err, however if you wan't to keep iterating even after failing to find an ACL (i.e  `errNotFound` error) You can do 

```if err !=nil && err != libovsdbclient.ErrNotFound```